### PR TITLE
Update new project checklist

### LINKED ISSUE #4: Update new project guide
- Using project template
- Update dependency manager info (update for SPM?)
- Using brew installed SwiftLint
- Remove Funtaster info
- Automatic removal of branches on GitHub
- Reorder steps (1. Create repo from template)

### LINKED ISSUE #6: Merge with Android and QA handbooks
Merge this project with Android and QA handbooks and create unufied "Engineering handbook".

- https://github.com/thefuntasty/android-handbook
- https://github.com/thefuntasty/qa-handbook

### PR DESCRIPTION
**Pull request by mkj-is**
- Almost all the points were outdated.
- SwiftLint configuration is now located in project template.
- Bitrise configuration was moved here from the wiki.
- Checklist is now structured like a checklist.

Closes #4.

### CODE REVIEW

**Comment by mkj-is** [comment]
Readable version can be found here:

https://github.com/thefuntasty/ios-handbook/blob/feature/project-checklist/new-project.md

File: new-project.md [hunk 1]
```diff
@@ -1,77 +1,101 @@
...
-`bundle exec fastlane match enterprise -a com.thefuntasty.PROJECT.beta -u ops@thefuntasty.com -b YVMX5P3692`
+## 5. Configure SwiftLint
 
-In Xcode, go to Project Info screen. In Configurations section, add a new configuration by duplicating the Release configuration and name it `Enterprise`. On Build Settings tab, look for Product Bundle Identifier, expand it to see preferences for Debug, Enterprise and Release configurations and for the Enterprise one, add suffix `.beta`.
+- [ ] Add a new Run Script Phase that executes the script:
 
-In Target's General tab, keep the Automatically manage signing checkbox unchecked and select appropriate certificate in the Signing (Enterprise) section. The certificate should be already there, if not, make sure Enterprise certificate and provisioning profile was correctly installed using the Fastlane Match.
+```
+if which swiftlint >/dev/null; then
+    swiftlint
```
Comment (bezoadam) [inline new-project.md]: Shouldn't we also use `swiftlint autocorrect` ?

File: new-project.md [hunk 2]
```diff
@@ -1,77 +1,101 @@
...
-## 1. Create a new Xcode project
+## 1. Create new repository
 
-Start with the "Single View App" template. Make sure Organization Name is *The Funtasty, s.r.o.* and Organization Identifier is *com.thefuntasty*. Leave *Use Core Data* unchecked and *Include Unit Tests* and *Include UI Tests* checked.
+- [ ] Open [iOS project template](https://github.com/thefuntasty/iOS-project-template) and use it to create new repository. Preferred repo name is *[productname]-[platform]*, where *[productname]* includes only name of the product, without client name (like Skoda).
+- [ ] Create `develop` branch and set is as a default branch.
+- [ ] Disable **Wiki**, **Issues**, **Projects** features we do not use.
+- [ ] Enable **Automatically delete head branches** so there is no need to delete branches manually after each PR merge.
+- [ ] Add labels:
+	- Do not review – #0e8a16
```
Comment (skywall) [inline new-project.md]: Not needed anymore.

File: new-project.md [hunk 3]
```diff
@@ -1,77 +1,98 @@
...
+## 1. Create new repository
 
-Start with the "Single View App" template. Make sure Organization Name is *The Funtasty, s.r.o.* and Organization Identifier is *com.thefuntasty*. Leave *Use Core Data* unchecked and *Include Unit Tests* and *Include UI Tests* checked.
+- [ ] Open [iOS project template](https://github.com/thefuntasty/iOS-project-template) and use it to create new repository. Preferred repo name is *[productname]-[platform]*, where *[productname]* includes only name of the product, without client name (like Skoda).
+- [ ] Create `develop` branch and set is as a default branch.
+- [ ] Disable **Wiki**, **Issues**, **Projects** features we do not use.
+- [ ] Enable **Automatically delete head branches** so there is no need to delete branches manually after each PR merge.
 
-## 2. Configure Ruby environment and Fastlane
+![GitHub labels](attachments/GitHub_labels.png)
```
Comment (skywall) [inline new-project.md]: !

**Pull request merged**